src/Helper/Navigation/AbstractHelper.php

```diff
@@ -14,6 +13,7 @@
 use Laminas\View;
 use Laminas\View\Exception;
 use Laminas\View\Helper\TranslatorAwareTrait;
+use Psr\Container\ContainerInterface;
```

Comment:
This will require us to bump this package to PHP 7.4 as we do change the argument type of `AbstractHelper#setServiceLocator` here.

This is only possible as of PHP 7.4 AFAIR, maybe @Ocramius has more insights here.

Reply:
I think that the bump to service manager ^3.4 solves that… Maybe…